doc/running-docker-compose.md

```diff
@@ -0,0 +1,38 @@
+# Running PrairieLearn + PrairieGrader with `docker-compose`
+
+If you need to develop or test [externally-graded questions](externalGrading.md), you should use this method of running PrairieLearn. This method will spin up PrairieLearn, PrairieGrader, and several other services, and ensure they are connected properly to enable external grading to work.
+
+## Instructions
+
+1. Install [Docker Community Edition](https://www.docker.com/community-edition). It's free.
```

Comment:
Could probably replace the first few points here with `* Install Docker (see the [PrairieLearn Docker instructions] for details)", to avoid duplication.